### PR TITLE
Automated backups

### PR DESCRIPTION
**Pull request by mbround18**
# Description

Implementation of #96 

## Contributions
- Cleaned up script files a bit.
- Added versatility for more cron jobs as needed.
- Renamed `init` to be `configure` based on what it does. 
- Added backup command which can tar.gz a directory
- Added cron for auto backups
- Updated read me <3
- Refactored logging to track multiple files based on their tasks. 

## Checklist

- [x] I added one or multiple labels which best describes this PR.
- [x] I have tested the changes locally.
- [x] This PR has a reviewer on it. 
- [x] I have validated my changes in a docker container and on Ubuntu. (Only needed for Odin or Docker Changes)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN apt-get update                  \
     htop net-tools nano             \
     netcat curl wget                \
     cron sudo gosu dos2unix         \
-    libsdl2-2.0-0  jq               \
+    libsdl2-2.0-0  jq  multitail    \
```
Comment (mbround18) [inline Dockerfile:16]: Needs to remove this

File: README.md [hunk 2]
```diff
@@ -10,22 +10,28 @@
...
+| WORLD                    | `Dedicated`            | TRUE     | This is used to generate the name of your world. |
+| PUBLIC                   | `1`                    | FALSE    | Sets whether or not your server is public on the server list. |
+| PASSWORD                 | `12345`                | TRUE     | Set this to something unique! |
+| AUTO_UPDATE              | `0`                    | FALSE    | Set to `1` if you want your container to auto update! This means at 1 am it will update, stop, and then restart your server. |
+| AUTO_UPDATE_SCHEDULE     | `0 1 * * *`            | FALSE    | This works in conjunction with `AUTO_UPDATE` and sets the schedule to which it will run an auto update. [If you need help figuring out a cron schedule click here]
+| AUTO_BACKUP              | `0`                    | FALSE    | Set to `1` to enable auto backups. Backups are stored under `/home/steam/backups` which means you will have to add a volume mount for this directory. |
+| AUTO_BACKUP_SCHEDULE     | `*/15 * * * *`         | FALSE    | Change this value to update based on how frequently you would like the server to update. [If you need help figuring out a cron schedule click here].
```
Comment (kpengwin) [inline README.md:26]: "how frequently you would like the server to update." should probably be "how frequently you would like the server to backup.

File: src/cli.yaml [hunk 3]
```diff
@@ -69,3 +69,16 @@ subcommands:
...
+      about: Backups the current saves to a specifc location
+      version: "1.0"
+      author: mbround18
+      args:
+        - INPUT:
+            about: Directory to backup
+            required: true
+            index: 1
+        - OUTPUT:
+            about: Sets the input file to use
```
Comment (LovecraftianHorror) [inline src/cli.yaml:82]: Nit, but should this be something along the lines of filename or filepath for the gzipped backup?

Comment (mbround18) [reply src/cli.yaml:82]: Yeah, I can update the about section

**Pull request merged**